### ISSUE TITLE
Build error: drawsurface.h:594:33: error: expected ')' before 'const'

### ISSUE THREAD

**Issue by stefanhusmann** [vectorgraphics/asymptote#275]
This happens under Arch Linux with latest sources from Git.

[buildlog.txt](https://github.com/vectorgraphics/asymptote/files/7470461/buildlog.txt)

**Comment by johncbowman** [comment]
That likely means you either have configured with --disable-gl or you are missing criticial OpenGL/glm libraries.
I will add a guard against this but you still have an issue to fix at your end.

**Comment by stefanhusmann** [comment]
Yes, --disable-gl was given to the configure script. If I add it, I get a different error.
```
glrender.cc: In function 'void gl::glrender(const string&, const camp::picture*, const string&, double, double, double, double, const camp::triple&, const camp::triple&, const camp::pair&, const camp::pair&, double*, double*, size_t, camp::triple*, double*, double*, bool, int)':
glrender.cc:1701:7: error: 'webgl' was not declared in this scope
 1701 |   if(!webgl) {
      |       ^~~~~
make: *** [Makefile:372: glrender.o] Error 1
```

**Comment by johncbowman** [comment]
Fixed in bf21a3428f7dcf3466fa78a10ab01b1c01fa8486
